app/dashboard/templates/bulk_payout_bounty.html

```diff
@@ -120,13 +121,52 @@ <h5>{% trans 'Payout Preview' %}</h5>
                     </a>
                   </div>
                 </div>
+                
+              </div>
+              <div id="tooltiphov">
+                    <span id="heading">Where is my Eth Going?</span><br>
```

Comment:
Can you wrap all of the copy throughout this file in `{% trans "copy" %}`

So here:  `{% trans "Where is my Eth Going?" %}`

Reply:
@usmanmuhd Just the text